packages/web3-utils/package.json

```diff
@@ -48,6 +48,7 @@
 	},
 	"dependencies": {
 		"ethereum-cryptography": "^1.1.2",
+		"ethereumjs-util": "^7.1.5",
```

Comment:
can we replace all usage of ethereumjs-util with our utility functions?
 ethereumjs-util bundles and brings so many unsused dependencies.